build/common.sh

```diff
@@ -875,7 +875,10 @@ function kube::release::write_addon_docker_images_for_server() {
         kube::log::status "Pulling and writing Docker image for addon: ${addon_path}"
 
         local dest_name="${addon_path//\//\~}"
-        "${DOCKER[@]}" pull "${addon_path}"
+        if [[ ! `${DOCKER[@]} images | awk '{print ($1":"$2)}' | grep ${addon_path}` ]]; then
```

Comment:
Use `-z` instead of `!` to check string length > 0.

Prefer `$(command)` to backticks.

Please quote `"${DOCKER[@]}"` and `"${addon_path}"`.